### PR TITLE
squoosh-v2 blog post

### PR DESCRIPTION
**Pull request by kosamari**
This is a blog post for squoosh v2.

### CODE REVIEW

File: src/site/content/en/blog/squoosh-v2/index.md
Comment (kaycebasques) [inline src/site/content/en/blog/squoosh-v2/index.md:6]:
```suggestion
```
`scheduled` doesn't work anymore. After I approve, please merge this PR when you want it to go live. We deploy the site every hour, and the deploy process takes about 20 minutes.

File: src/site/content/en/blog/squoosh-v2/index.md
Comment (kaycebasques) [inline src/site/content/en/blog/squoosh-v2/index.md:3]:
```suggestion
subhead: New codecs support, updated design, and CLI support!
```

File: src/site/content/en/blog/squoosh-v2/index.md
Comment (kaycebasques) [inline src/site/content/en/blog/squoosh-v2/index.md:5]:
```suggestion
  New codecs support, updated design, and CLI support!
```
It's OK for the `description` to duplicate the `subhead` and in this case I think the `subhead` is more useful

File: src/site/content/en/blog/squoosh-v2/index.md
Comment (kaycebasques) [inline src/site/content/en/blog/squoosh-v2/index.md:9]:
```suggestion
alt: A screen capture of the new Squoosh homepage
```

File: src/site/content/en/blog/squoosh-v2/index.md
Comment (kaycebasques) [inline src/site/content/en/blog/squoosh-v2/index.md:19]: Please break all lines at 100 characters. If you're using VS Code, [Rewrap](https://marketplace.visualstudio.com/items?itemName=stkb.rewrap) makes it pretty easy to do. The [Docs To X](https://gsuite.google.com/u/0/marketplace/app/docs_to_x/243920501110?hl=en&pann=docs_addon_widget) Google Docs Add-On also supports this.

File: src/site/content/en/blog/squoosh-v2/index.md
Comment (kaycebasques) [inline src/site/content/en/blog/squoosh-v2/index.md:7]:
```suggestion
date: 2020-12-09
authors:
  - kosamari
```
We need you listed as author so that when people read "Squoosh is an app that our team built" they know who is talking

File: src/site/content/en/blog/squoosh-v2/index.md
Comment (kaycebasques) [inline src/site/content/en/blog/squoosh-v2/index.md:8]: Is this how you want it to appear on web.dev/blog? Except for the `draft` thing (you can ignore that)

image

File: src/site/content/en/blog/squoosh-v2/index.md
Comment (kaycebasques) [inline src/site/content/en/blog/squoosh-v2/index.md:26]:
```suggestion
We now support OxiPNG, MozJPEG, WebP, and AVIF, in addition to codecs with built-in support in your browser.  A new codec was made possible again with the use of WebAssembly. By compiling a codec encoder and decoder as a WebAssembly module users can access and experiment with newer codecs even if their preferred browser does not support them. 
```
"native" is on P&E's "do not use" word list because it has some bad political assocations

File: src/site/content/en/blog/squoosh-v2/index.md
Comment (kaycebasques) [inline src/site/content/en/blog/squoosh-v2/index.md:28]:
```suggestion
```
I don't like to make promises in docs because we (as a general team) often end up not delivering on our promises. So if the doc isn't ready by the time we publish this, then I do not want to mention it

File: src/site/content/en/blog/squoosh-v2/index.md
Comment (kaycebasques) [inline src/site/content/en/blog/squoosh-v2/index.md:33]:
```suggestion
Ever since the original launch in 2018, a common user request was to interact with Squoosh programmatically without a GUI. We felt a bit conflicted about this path since our app was a UI on top of command line based codec tools. However we do understand the desire to interact with the whole package of codecs instead of multiple tools. Squoosh CLI does just that.
```

File: src/site/content/en/blog/squoosh-v2/index.md
Comment (kaycebasques) [inline src/site/content/en/blog/squoosh-v2/index.md:39]:
```suggestion
The Squoosh CLI is written in Node and makes use of the exact same WebAssembly modules the PWA uses. Through extensive use of workers, all images are decoded, processed and encoded in parallel. We also use Rollup to bundle everything into one JavaScript file to make sure installation via `npx` is quick and seamless. The CLI also offers auto compression, where it tries to reduce the quality of an image as much as possible without degrading the visual fidelity (using the [Butteraugli metric](https://github.com/google/butteraugli)). 
```
We follow MDN's formatting of names, and MDN does not generally capitalize "workers"

File: src/site/content/en/blog/squoosh-v2/index.md
Comment (kaycebasques) [inline src/site/content/en/blog/squoosh-v2/index.md:41]:
```suggestion
With the Squoosh CLI you can compress the images in your web app to multiple formats and use the [`<picture>` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/picture#The_type_attribute) to let the browser choose the best version. We also plan to build plugins for Webpack, Rollup, and other build tools to make image compression a part of your build step that happens automatically.
```

File: src/site/content/en/blog/squoosh-v2/index.md
Comment (kaycebasques) [inline src/site/content/en/blog/squoosh-v2/index.md:46]: Please review the rest of the doc and make sure you're capitalizing Webpack and Rollup everywhere

File: src/site/content/en/blog/squoosh-v2/index.md
Comment (kaycebasques) [inline src/site/content/en/blog/squoosh-v2/index.md:48]:
```suggestion
The project was initially started with webpack because we wanted to try it as a team, and at the time in 2018 webpack was the only tool that gave us enough control to set up the project the way we wanted. Over time, we've found Rollup's easy plugin system and simplicity with ESM made it a natural choice for this project. 
```

File: src/site/content/en/blog/squoosh-v2/index.md
Comment (kaycebasques) [inline src/site/content/en/blog/squoosh-v2/index.md:53]:
```suggestion
We've also updated the UI design of the app featuring `blobs` as a visual element.  It is a little pun on how we treat data in our code. Squoosh passes image data around as a blob, so it felt natural to include some blobs in the design(get it?).
```

File: src/site/content/en/blog/squoosh-v2/index.md
Comment (kaycebasques) [inline src/site/content/en/blog/squoosh-v2/index.md:62]:
```suggestion
Squoosh has always been open source but we've never had focus on growing the community. In 2021, we plan to expand our contributor base and have a better onboarding process to the project. 
```
Plz add link to the repo

File: src/site/content/en/blog/squoosh-v2/index.md
Comment (kaycebasques) [inline src/site/content/en/blog/squoosh-v2/index.md:64]:
```suggestion
Do you have any ideas for Squoosh? Please let us know on our issue tracker. The team is headed to extended winter vacation but we promise to get back to you in the new year.
```
Link to the issue tracker

**Review by kaycebasques — CHANGES_REQUESTED** [review]
@kosamari please resolve the changes that I've requested in the comments and then mention me in a comment when it's ready for another review

**Comment by kosamari** [comment]
@kaycebasques PTAL :)

**Comment by argyleink** [comment]
that thumbnail really got compressed...?



vs 

thumb – 3

**Comment by kosamari** [comment]
@argyleink joke is on me! I squoosh-ed Squoosh header and used same setting for thumnail which made it look bad. just updated to better compression setting jpeg.

File: src/site/content/en/blog/squoosh-v2/index.md
Comment (kaycebasques) [inline src/site/content/en/blog/squoosh-v2/index.md:32]:
```suggestion
browser.  A new codec was made possible again with the use of WebAssembly. By compiling a codec 
```

File: src/site/content/en/blog/squoosh-v2/index.md
Comment (kaycebasques) [inline src/site/content/en/blog/squoosh-v2/index.md:39]:
```suggestion
```

File: src/site/content/en/blog/squoosh-v2/index.md
Comment (kaycebasques) [inline src/site/content/en/blog/squoosh-v2/index.md:44]:
```suggestion
top of command-line-based codec tools. However we do understand the desire to interact with the 
```

File: src/site/content/en/blog/squoosh-v2/index.md
Comment (kaycebasques) [inline src/site/content/en/blog/squoosh-v2/index.md:53]:
```suggestion
uses. Through extensive use of workers, all images are decoded, processed and encoded in parallel. 
```

File: src/site/content/en/blog/squoosh-v2/index.md
Comment (kaycebasques) [inline src/site/content/en/blog/squoosh-v2/index.md:60]:
```suggestion
[`<picture>` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/picture) 
```

File: src/site/content/en/blog/squoosh-v2/index.md
Comment (kaycebasques) [inline src/site/content/en/blog/squoosh-v2/index.md:64]:
```suggestion
plugins for Webpack, Rollup, and other build tools to make image compression
an automatic part of your build process.
```

File: src/site/content/en/blog/squoosh-v2/index.md
Comment (kaycebasques) [inline src/site/content/en/blog/squoosh-v2/index.md:82]:
```suggestion
natural to include some blobs in the design (get it?).
```

File: src/site/content/en/blog/squoosh-v2/index.md
Comment (kaycebasques) [inline src/site/content/en/blog/squoosh-v2/index.md:99]:
```suggestion
Do you have any ideas for Squoosh? Please let us know on our
[issue tracker](https://github.com/GoogleChromeLabs/squoosh/issues).
The team is headed to extended winter vacation but we promise to get
back to you in the new year.
```

**Review by kaycebasques — CHANGES_REQUESTED** [review]
Minor edits that I will commit myself

**Review by kaycebasques — APPROVED** [review]

**Comment by kaycebasques** [comment]
@kosamari merge the PR when you're ready. We deploy the site every hour, and the deploy process takes ~20 minutes.

**Pull request merged**
